addons/event/static/src/client_action/event_barcode.scss

```diff
@@ -24,6 +24,35 @@
         max-width: 200px;
         max-height: 100px;
     }
+    .o_barcode_mobile_container {
+        margin-top: 75px;
+        margin-bottom: -35px;
+
+        img {
+            height: 185px;
+            width: 275px;
+        }
+        .o_mobile_barcode {
+            opacity: 0;
+            height: 220px;
+            width: 275px;
+            bottom: -35px;
+        }
+        .o_barcode_laser {
+            height: 3px;
+        }
+    }
+    @include media-breakpoint-down(md) {
+        padding: 0 1em 1em .75em;
+
+        .o_tap_to_scan {
+            display: flex;
+            flex-direction: column;
+            justify-content: center;
+            align-items: center;
```

Comment:
actually we have selectors for all this, it is better to use them directly on the div rather than in the css.
(This is a generic comment, maybe other places in the PR, to check):

justify-content-center, flex-column, d-flex...